android_docker_image.md

```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
+optimized for CI/CD. It comes with a set of preinstalled utility tools commonly
+used for CI/CD workflows.
+
+[[__TOC__]]
+
+## Using the semaphoreci/android docker image in your semaphore.yml configuration
+
+To use the images in this repository for your CI/CD pipelines, set up your Semaphore YAML with:
+
+``` yaml
+version: v1.0
+name: Hello andorid
+agent:
+  machine:
+    type: e1-standard-2
+  containers:
+    - name: main
+      image: semaphoreci/android:28
+blocks:
+  - name: "Hello"
+    task:
+      jobs:
+      - name: Hello
+        commands:
+          - checkout
+          - echo "Hello"
+```
+
+## Dockerhub repository
+The full list of android docker containers can be found under [SemaphoreCI docker registry](https://hub.docker.com/repository/docker/semaphoreci/android).
+
+## Dockerfiles
+Dockerfiles used to build an android image can be found under [Docker images repository](https://github.com/semaphoreci/docker-images/tree/master/dockerfiles/android).
+
+## Image Variants
+
+The android image.
+
+ - andorid:version is the basic image that comes with development utilities and is an optimal first choice for andorid.
```

Comment:
Typo, use `code`

Reply:
Revise this section after you see how the new structure looks as per earlier comments.